app/_redirects

```diff
@@ -40,10 +40,8 @@
 /gateway/latest/kong-enterprise/analytics/* /gateway/3.4.x/kong-enterprise/analytics/:splat
 /gateway/latest/kong-enterprise/dev-portal/* /gateway/3.4.x/kong-enterprise/dev-portal/:splat
 /gateway/latest/how-kong-works/performance-testing/ /gateway/3.4.x/how-kong-works/performance-testing/
-/gateway/3.5.x/how-kong-works/performance-testing/ /gateway/3.4.x/how-kong-works/performance-testing/
```

Comment:
I'm guessing you didn't mean to remove these, so I'm adding them back.

Reply:
Correct